src/modules/auto-toggle-class-on-scroll.js

```diff
@@ -106,8 +106,8 @@
 	var scrollTimer = null;
 	
 	var postscroll = function () {
-		global.craf(scrollTimer);
-		scrollTimer = global.raf(function () {
+		window.craf(scrollTimer);
+		scrollTimer = window.raf(function () {
```

Comment:
the signature of the main function is not similar between files. I prefer for the moment using window as the standard and in a second step unifying all main function